DP-38918: Add new content type for API Service Card

joeg8612 opened the pull request:
**Description:**
Explain the technical implementation of the work done.


**Jira:** (Skip unless you are MA staff)
DP-38918


**To Test:**
- [ ] Add steps to test this feature


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

Commit:
new content type

arthurbaghdas commented:
<!--
  1 failure:  Add a changelog Y...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against dd3f3dfdaf6fefe7e87660c7a932b2d7df48f53e
</p>

dstorozhuk committed:
DP-38918: Add API Service Card bundle and content moderation workflow

Introduced the `ApiServiceCardBundle` class for managing node entities and updated the `mass_content_entity_bundle_info_alter()` function to register it. Added a content moderation workflow specific to API Service Cards and configured form displays, field groupings, and weights. Additionally, updated language settings and field properties to align with requirements.

Signed-off-by: Dmytro Storozhuk <dima@itech4web.com>

dstorozhuk committed:
DP-38918: Add view configuration for API Service Cards

Introduced a new view configuration to list and manage API Service Cards. The view includes both administrative and default displays with fields like Title, Created, and Changed, and is accessible via a dedicated admin page at "admin/content/api-service-cards".

Signed-off-by: Dmytro Storozhuk <dima@itech4web.com>

Commit:
view enhancements

dstorozhuk committed:
DP-38918: Update API Service Cards view and add MMG Editor role

Renamed menu and updated paths in the API Service Cards view to align with the mass dashboard. Introduced a new MMG Editor role with specific permissions and actions for assigning/removing the role from users. Adjusted language type configurations for consistency.

Signed-off-by: Dmytro Storozhuk <dima@itech4web.com>

dstorozhuk commented:
closed in favor of https://github.com/massgov/openmass/pull/2976 .
